db/migrate/20250403155909_add_session_date_id_to_pre_screenings.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class AddSessionDateIdToPreScreenings < ActiveRecord::Migration[6.1]
+  def up
+    add_reference :pre_screenings, :session_date, foreign_key: true, null: true
+
+    PreScreening.reset_column_information
```

Comment:
I don't think you need to call `reset_column_information` because you're using `update_column` which bypasses validations and callbacks, working directly with the database. `reset_column_information `.